### ISSUE TITLE
Improving NFC Reading Speed for Turkey Identity Verification

### ISSUE THREAD

**Issue by erdiergene** [AndyQ/NFCPassportReader#228]
We are using your NFC library for Turkey's identity verification, retrieving information such as the photo, date of birth, and expiry date. However, the NFC reading process is a bit slow. Do you have any recommendations to speed it up?

**Comment by Melsaeed276** [comment]
Hello @erdiergene I am using it too and somehow it works fine with me. yes it is a bit slow but you might need to remove the tags that you do not need to read, in this case it might be faster for you.

**Comment by AndyQ** [comment]
Yeah retrieving images can be slow - depending on the passport and features it supports.  Sadly at the moment there is not much I can do about this I don't think.
